README.md

```diff
@@ -1,58 +1,83 @@
-<br />
-<div align="center">
-  <img src="https://github.com/kevinadhiguna/wifi-sploit/blob/master/assets/wifi-sploit.png" />
-  <h3 align="center">Wi-Fi Sploit</h3>
-
-  <p align="center">
-   🔒 A password cracker for an admin page of a Wi-Fi router
-  </p>
+<div style="display: flex; justify-content: space-between;">
+    <img src="https://github.com/d4v1-sudo/wifi-sploit/raw/master/assets/router.jpg" style="text-align: left ; height:25%; width:25%" />
+    <img src="https://github.com/d4v1-sudo/wifi-sploit/raw/master/assets/slash.png" style="text-align: center ; height:25%; width:25%" />
+    <img src="https://github.com/d4v1-sudo/wifi-sploit/raw/master/assets/www.jpg" style="text-align: right ; height:25%; width:25%" />
 </div>
+<h3 style="text-align:center;">WiFi-Sploit</h3>
+<p style="text-align:center;">
+   🔒 A password cracker for any login page
+</p>
 
-<br />
-
-## Status
+## About
 
-🚧 Currently under development, originally created using Python 2.7 but trying to make it compatible with Python 3.
-
-<br />
+ - Originally maded for `python2` but it runs using `pyhon3` too.
```

Comment:
Let me change this typo (`maded` to `made`) 

Thank you for providing descriptions for each script, much appreciated :)

Reply:
Ok, nice. I'm glad you liked it.